src/components/network-chart/index.css

```diff
@@ -54,23 +54,85 @@
 }
 
 .networkChartRowItemBar {
-  background-color: var(--blue-50);
+  background-color: white;
+  border: 1px solid var(--grey-40);
   display: inline-block;
   z-index: 3;
   border-radius: 2px;
   height: 12px;
   flex-grow: 1;
   position: absolute;
   margin-top: 2px;
+  opacity: 0.7;
+}
+
+.networkChartRowItemBarInner {
+  height: 12px;
+  display: inline-block;
+  background-color: var(--grey-40);
+}
+
+.css .networkChartRowItemBar {
```

Comment:
`.css`, `.js`, and `.html` are really broad selectors. I think it would be better to make them more specific to the component, and have them match the class name system we have in place, so something like `.networkChartRowItemCss`